__tests__/feature/editing/addRemove.test.js

```diff
@@ -20,7 +20,7 @@ describe('adding and removing properties', () => {
     await screen.findByText('Uber template1', { selector: 'h3' })
 
     // Add a panel property
-    expect(screen.getAllByText(/Uber template1, property2/)).toHaveLength(2)
+    expect(screen.getAllByText(/Uber template1, property2$/)).toHaveLength(2) // do not want to match property20!
```

Comment:
had to modify this matcher because it was incorrectly picking up our new property20 field

Reply:
Thanks! Good change.